packages/@mantine/core/src/components/Combobox/Combobox.tsx

```diff
@@ -159,7 +160,7 @@ export function Combobox(_props: ComboboxProps) {
       <Popover
         opened={store.dropdownOpened}
         {...others}
-        onClose={store.closeDropdown}
+        onClose={onClose || store.closeDropdown}
```

Comment:
I suppose that it would be better to call both functions:

`() => {onClose?.(); store.closeDropdown()}`

Reply:
I agree it's a good idea to call both functions. I've updated it to trigger both close handlers